ucp/_libs/utils.pyx

```diff
@@ -1,38 +1,40 @@
 # Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
 # See file LICENSE for terms.
 # cython: language_level=3
-import asyncio
-import operator
-from functools import reduce
 
 from cpython.memoryview cimport PyMemoryView_GET_BUFFER
-from libc.stdint cimport uintptr_t
+from libc.stdint cimport int64_t, uint64_t, uintptr_t
+
+from array import array
 
 from ..exceptions import UCXCloseError, UCXError
 
 
-def get_buffer_data(buffer, check_writable=False):
+cpdef uintptr_t get_buffer_data(buffer, bint check_writable=False) except *:
```

Comment:
```suggestion
cpdef uintptr_t get_buffer_data(object buffer, bint check_writable=False) except *:
```

Reply:
Cython already defaults to `object` if a type is specified. So we are ok here.